docs/content/docs/try-flink/local_installation.md

```diff
@@ -24,71 +24,139 @@ specific language governing permissions and limitations
+
+{{< hint info >}}
+__Note:__ Flink is also available as a [Docker image](https://hub.docker.com/_/flink).
 {{< /hint >}}
-{{< /unstable >}}
 
-Follow these few steps to download the latest stable versions and get started.
+Flink runs on all UNIX-like environments, i.e. Linux, Mac OS X, and Cygwin (for Windows). You need 
+to have __Java 8 or 11__ installed. To check the Java version installed, type in your terminal: 
 
-## Step 1: Download
+```bash
+$ java -version
+```
 
-To be able to run Flink, the only requirement is to have a working __Java 8 or 11__ installation.
-You can check the correct installation of Java by issuing the following command:
+Next, [download the latest binary release](https://flink.apache.org/downloads.html) of Flink, 
+then extract the archive: 
 
 ```bash
-java -version
+$ tar -xzf flink-*.tgz
 ```
 
-[Download](https://flink.apache.org/downloads.html) the {{< version >}} release and un-tar it. 
+## Browsing the project directory
+
+Navigate to the extracted directory and list the contents by issuing:
 
 ```bash
-$ tar -xzf flink-{{< version >}}-bin-scala{{< scala_version >}}.tgz
-$ cd flink-{{< version >}}-bin-scala{{< scala_version >}}
+$ cd flink-* && ls -l
 ```
 
-## Step 2: Start a Cluster
+You should see something like:
+
+![project directory](projectdirectory.png)
+
+For now, you may want to note that:
+- __bin/__ directory contains the `flink` binary as well as several bash scripts that manage various jobs and tasks
+- __conf/__ directory contains configuration files, including `flink-conf.yaml`
+- __examples/__ directory contains sample applications that can be used as is with Flink
 
-Flink ships with a single bash script to start a local cluster.
+
+## Starting and stopping a local cluster
+
+To start a local cluster, run the bash script that comes with Flink:
 
 ```bash
 $ ./bin/start-cluster.sh
-Starting cluster.
-Starting standalonesession daemon on host.
-Starting taskexecutor daemon on host.
 ```
 
-## Step 3: Submit a Job
+You should see an output like this:
+
+![output](output.png)
+
+Flink is now running as a background process. You can check its status with the following command:
+
+```bash
+$ ps aux | grep flink
+```
+
+You should be able to navigate to the web UI at [localhost:8081](http://localhost:8081) to view
+the Flink dashboard and see that the cluster is up and running. 
+
+To quickly stop the cluster and all running components, you can use the provided script:
+
+```bash
+$ ./bin/stop-cluster.sh
+```
+
+## Submitting a Flink job
 
-Releases of Flink come with a number of example Jobs.
-You can quickly deploy one of these applications to the running cluster. 
+Flink provides a CLI tool, __bin/flink__, that can run programs packaged as Java ARchives (JAR)
+and control their execution. Submitting a [job](https://ci.apache.org/projects/ﬂink/ﬂink-docs-stable/concepts/glossary.html#ﬂink-job) means uploading the job’s JAR ﬁle and related dependencies to the running Flink cluster
```

Comment:
When cross-linking to other pages in the docs, please use `ref` syntax. Hugo will fail the build if the link is broken, it also ensures links are properly versioned. 

https://gohugo.io/content-management/cross-references/

```suggestion
and control their execution. Submitting a [job]({{< ref "docs/concepts/glossary" >}}#ﬂink-job) means uploading the job’s JAR ﬁle and related dependencies to the running Flink cluster
```